.chloggen/datadog-trace-buffer.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Add a new traces config `trace_buffer` that specifies the number of Datadog Agent TracerPayloads to buffer before dropping."
```

Comment:
I would rewrite this in a way that is more user-focused. Most users don't know what a Datadog Agent TracerPayload is, so this may be difficult to understand.

Reply:
SG, rephrased a bit.